src/Components/Footer/Footer.scss

```diff
@@ -1,9 +1,17 @@
-.footer-logo-container {
+.footer {
   background-color: #3c3525;
   padding: 1.5rem;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  flex-wrap: wrap;
+
+  .logo-img {
+    height: 4rem;
+    filter: invert(1);
```

Comment:
That logo warms my heart! ❤️

Reply:
That filter: invert is sweet!! Essentially all that does is change the logo from black to white! I found it in a stack overflow article I can share with yall tomorrow.